### ISSUE TITLE
Error: closing tag mismatch

### ISSUE THREAD

**Issue by Bartheyrman22** [atSCM/atscm#330]
During atscm pull the process exits with an error message.

**Steps to reproduce error**

 - *Use beta version 41*
 - *try to pull HEADER.header (annex)*

**Original error message**

![image](https://user-images.githubusercontent.com/12160269/65326313-73ccb280-dbb1-11e9-8b76-6c4a11f37a88.png)
 
---

<details><summary><strong>Environment</strong></summary>

| package       |               version |
| :------------ | --------------------: |
| OS            | *Windows* |
| atscm         |         1.0.0-beta.41 | 
| atscm-cli     |     0.8.2-beta.1 | 
| atvise server | 3.3.4 |
| node          |          10.16.3 | 
| npm           |           6.11.3 | 
 
</details>

<details><summary><strong>Project configuration</strong></summary>

```javascript
// Atviseproject.babel.js
import { Atviseproject } from 'atscm';

/**
 * atvise-scm configuration of atscm-project.
 */
export default class AtscmProject extends Atviseproject {

  /**
   * The atvise-server's host
   * @type {String}
   */
  static get host() {
    return 'localhost';
  }

  /**
   * The atvise-server ports to use.
   * @type {Object}
   * @property {Number} opc The OPC-UA port the atvise-server runs on.
   * @property {Number} http The HTTP port the atvise-server can be reached at.
   */
  static get port() {
    return {
      opc: 4840,
      http: 80,
    };
  }

  /**
   * The login to use. Return `false` if no login is required.
   * @type {Boolean|Object}
   * @property {String} username The username to log in with.
   * @property {String} password The password to log in with.
   */
  static get login() {
    return {
      username: '',
      password: '',
    };
  }

}
```

**Code generating error:**

```xml
<svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" atv:refpx="1639" width="80" atv:refpy="30" y="0" id="id_9" x="1599">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.alarm"/>
  <atv:argument name="text" value="T{ALARM}"/>
  <atv:overwrite d="M7.658,1.148L7.117,1.3L8,4.438L8.542,4.284ZM12.573,1.148L11.688,4.284L12.23,4.438L13.113,1.3ZM3.799,3.541L3.41,3.93L5.662,6.18L6.051,5.791ZM16.432,3.541L14.179,5.791L14.566,6.18L16.82,3.93ZM9.828,6.562C9.788,6.563 9.749,6.567 9.71,6.57C9.571,6.581 9.433,6.599 9.297,6.619C9.16,6.643 9.026,6.671 8.894,6.703C8.76,6.736 8.63,6.774 8.502,6.817C8.375,6.859 8.249,6.907 8.127,6.961C8.005,7.013 7.887,7.07 7.773,7.131C7.659,7.193 7.549,7.258 7.443,7.327C7.337,7.4 7.235,7.473 7.14,7.551C7.043,7.628 6.951,7.709 6.867,7.796C6.781,7.881 6.701,7.969 6.627,8.061C6.552,8.151 6.484,8.246 6.422,8.343C6.359,8.439 6.304,8.538 6.254,8.638C6.206,8.74 6.161,8.843 6.125,8.947C6.097,9.033 6.074,9.119 6.054,9.207L6.026,10.216L5.833,17.151L5,17.151L5,18.612L15.254,18.612L15.254,17.151L14.499,17.151L14.305,10.179L14.279,9.26C14.267,9.192 14.252,9.124 14.232,9.056C14.199,8.952 14.163,8.848 14.118,8.745C14.074,8.643 14.021,8.543 13.965,8.445C13.906,8.346 13.842,8.251 13.772,8.157C13.702,8.064 13.626,7.975 13.545,7.887C13.462,7.799 13.376,7.716 13.283,7.634C13.191,7.555 13.093,7.477 12.99,7.405C12.886,7.331 12.779,7.263 12.669,7.197C12.557,7.132 12.442,7.071 12.323,7.016C12.203,6.961 12.08,6.909 11.955,6.864C11.829,6.817 11.698,6.777 11.569,6.74C11.438,6.703 11.304,6.673 11.169,6.645C11.033,6.619 10.897,6.599 10.759,6.582C10.659,6.574 10.559,6.567 10.458,6.562ZM1.228,7.138L1.085,7.669L4.163,8.492L4.304,7.961ZM19.003,7.138L15.927,7.961L16.067,8.492L19.144,7.669Z" id="symbol>body"/>
  <atv:argument name="base" prefix="base"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.SYSTEM.Status.Alarm"/>
 </svg>
```

```xml
<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<svg version="1.2" height="60" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:atv="http://webmi.atvise.com/2007/svgext" width="1920" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:cc="http://creativecommons.org/ns#">
 <defs/>
 <metadata>
  <atv:parameter name="background" desc="Background" valuetype="global" group="Appearance" behavior="optional" substitute="$BACKGROUND$" config="SYSTEM.GLOBALS.Colors" defaultvalue="HeaderBackground"/>
  <atv:parameter name="homeDisplay" desc="Home Display" valuetype="global" group="Appearance" behavior="optional" substitute="$HOMEDISPLAY$" config="SYSTEM.GLOBALS.Settings" defaultvalue="homeDisplay"/>
  <atv:parameter name="map" desc="Enable map" valuetype="global" group="Appearance" behavior="hidden" substitute="$MAP$" config="SYSTEM.GLOBALS.Settings" defaultvalue="map"/>
  <atv:parameter name="active" desc="" valuetype="global" behavior="hidden" substitute="$ACTIVE$" config="SYSTEM.GLOBALS.Colors" defaultvalue="Active"/>
  <atv:parameter name="headertext" valuetype="global" behavior="hidden" substitute="$HEADERTEXT$" config="SYSTEM.GLOBALS.Colors" defaultvalue="HeaderText"/>
  <atv:parameter name="layers" valuetype="global" behavior="hidden" config="SYSTEM.GLOBALS.Appearance" defaultvalue="layers"/>
  <atv:gridconfig height="10" gridstyle="lines" width="10" enabled="true"/>
  <atv:snapconfig height="1" width="1" enabled="true"/>
 </metadata>
 <g transform="matrix(1.1999,0,0,0.3,0,0)" atv:insensitive="true" atv:refpx="0" atv:refpy="0" id="id_1">
  <rect fill="$BACKGROUND$" height="200.00" atv:refpx="800" width="1600.00" atv:refpy="50" y="0" id="id_4" x="0"/>
 </g>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.user" height="60" atv:refpx="330" width="80" atv:refpy="0" y="0" id="id_0" x="1840">
  <atv:argument name="base" prefix="base"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" atv:refpx="1560" width="80" atv:refpy="30" y="0" id="btnHome" x="1520">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.home"/>
  <atv:argument name="text" value="T{HOME}"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.Main"/>
  <atv:argument name="base" prefix="base"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" atv:refpx="1639" width="80" atv:refpy="30" y="0" id="id_9" x="1599">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.alarm"/>
  <atv:argument name="text" value="T{ALARM}"/>
  <atv:overwrite d="M7.658,1.148L7.117,1.3L8,4.438L8.542,4.284ZM12.573,1.148L11.688,4.284L12.23,4.438L13.113,1.3ZM3.799,3.541L3.41,3.93L5.662,6.18L6.051,5.791ZM16.432,3.541L14.179,5.791L14.566,6.18L16.82,3.93ZM9.828,6.562C9.788,6.563 9.749,6.567 9.71,6.57C9.571,6.581 9.433,6.599 9.297,6.619C9.16,6.643 9.026,6.671 8.894,6.703C8.76,6.736 8.63,6.774 8.502,6.817C8.375,6.859 8.249,6.907 8.127,6.961C8.005,7.013 7.887,7.07 7.773,7.131C7.659,7.193 7.549,7.258 7.443,7.327C7.337,7.4 7.235,7.473 7.14,7.551C7.043,7.628 6.951,7.709 6.867,7.796C6.781,7.881 6.701,7.969 6.627,8.061C6.552,8.151 6.484,8.246 6.422,8.343C6.359,8.439 6.304,8.538 6.254,8.638C6.206,8.74 6.161,8.843 6.125,8.947C6.097,9.033 6.074,9.119 6.054,9.207L6.026,10.216L5.833,17.151L5,17.151L5,18.612L15.254,18.612L15.254,17.151L14.499,17.151L14.305,10.179L14.279,9.26C14.267,9.192 14.252,9.124 14.232,9.056C14.199,8.952 14.163,8.848 14.118,8.745C14.074,8.643 14.021,8.543 13.965,8.445C13.906,8.346 13.842,8.251 13.772,8.157C13.702,8.064 13.626,7.975 13.545,7.887C13.462,7.799 13.376,7.716 13.283,7.634C13.191,7.555 13.093,7.477 12.99,7.405C12.886,7.331 12.779,7.263 12.669,7.197C12.557,7.132 12.442,7.071 12.323,7.016C12.203,6.961 12.08,6.909 11.955,6.864C11.829,6.817 11.698,6.777 11.569,6.74C11.438,6.703 11.304,6.673 11.169,6.645C11.033,6.619 10.897,6.599 10.759,6.582C10.659,6.574 10.559,6.567 10.458,6.562ZM1.228,7.138L1.085,7.669L4.163,8.492L4.304,7.961ZM19.003,7.138L15.927,7.961L16.067,8.492L19.144,7.669Z" id="symbol>body"/>
  <atv:argument name="base" prefix="base"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.SYSTEM.Status.Alarm"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" atv:refpx="1719" width="80" atv:refpy="20" y="0" id="id_10" x="1679">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.trend"/>
  <atv:argument name="text" value="T{TREND}"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.REPORT.Highchart"/>
  <atv:argument name="base" prefix="base"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" atv:refpx="1799" width="80" atv:refpy="20" y="0" id="id_3" x="1759">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.history"/>
  <atv:argument name="text" value="T{REPORT}"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.SYSTEM.REPORT.Heatmap"/>
  <atv:argument name="base" prefix="base"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.tabButton" height="60" atv:refpx="692" width="166" atv:refpy="29.032" y="0" id="id_6" x="606">
  <atv:argument name="tabListener" value="1"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.tabButton" height="60" atv:refpx="1194" width="166" atv:refpy="29.032" y="0" id="id_11" x="1109">
  <atv:argument name="text" value=""/>
  <atv:argument name="tabListener" value="4"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.tabButton" height="60" atv:refpx="861.667" width="166" atv:refpy="29.032" y="0" id="id_2" x="775.667">
  <atv:argument name="tabListener" value="2"/>
 </svg>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.tabButton" height="60" atv:refpx="1028.333" width="166" atv:refpy="29.032" y="0" id="id_8" x="942.333">
  <atv:argument name="tabListener" value="3"/>
 </svg>
 <g visibility="hidden" atv:refpx="314.18" atv:refpy="30.54" id="path">
  <text fill="$HEADERTEXT$" transform="matrix(1.295,0,0,1.295,-79.355,-4.807)" font-weight="bold" visibility="hidden" atv:refpx="274.828" font-family="Calibri" atv:refpy="29.893" font-size="15" y="31.295" id="site" x="273.5"/>
  <g atv:refpx="306.004" atv:refpy="30.54" id="buildingPoint">
   <text fill="$HEADERTEXT$" transform="matrix(1.295,0,0,1.295,-91.155,-4.425)" font-weight="bold" atv:refpx="314.827" font-family="Calibri" atv:refpy="29.893" font-size="15" y="31" id="building" x="313.5"/>
   <circle fill="$ACTIVE$" cy="30.541" cx="295.885" stroke="none" atv:refpx="309.873" r="3.238" stroke-width="0" atv:refpy="31.541" id="id_15"/>
  </g>
  <g atv:refpx="346.004" atv:refpy="30.54" id="floorPoint">
   <text fill="$HEADERTEXT$" transform="matrix(1.295,0,0,1.295,-102.955,-4.425)" font-weight="bold" atv:refpx="354.179" font-family="Calibri" atv:refpy="29.893" font-size="15" y="31" id="floor" x="353.5"/>
   <circle fill="$ACTIVE$" cy="30.541" cx="335.885" stroke="none" atv:refpx="425.775" r="3.238" stroke-width="0" atv:refpy="30.541" id="id_16"/>
  </g>
 </g>
 <svg xlink:href="AGENT.DISPLAYS.MENU.Logo" height="60" atv:refpx="132.5" width="265" atv:refpy="30" y="0" id="id_17" x="0"/>
 <svg xlink:href="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton" height="60" visibility="hidden" atv:refpx="1560" width="80" atv:refpy="30" y="0" id="btnMap" x="1520">
  <atv:argument name="symbol" value="SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.Symbols.20x20.location"/>
  <atv:argument name="text" value="T{MAP}"/>
  <atv:argument name="display" value="AGENT.DISPLAYS.MAIN.Map"/>
  <atv:argument name="base" prefix="base"/>
 </svg>
 <script type="text/ecmascript"><![CDATA[// Init and Param

});]]></script>
</svg>
```

</details>

**Comment by LukasHechenberger** [comment]
Besides, I'd strongly recommend you to not include `>`s in element ids, as it messes up css selectors: 

```javascript
// Selects an element with the id 'symbol>body'
document.querySelector('#symbol>body');

// Selects a child `<body>`-element of an element with the id 'symbol'
document.querySelector('#symbol > body');
```

**Comment by LukasHechenberger** [comment]
The parser already has an issue on this one https://github.com/nikku/saxen/issues/17

**Comment by LukasHechenberger** [comment]
Well, the component link tag has the id `id_9`, but the `<atv:overwrite>` element uses the nested component's id, so the element should be inside *SYSTEM.LIBRARY.PROJECT.OBJECTDISPLAYS.HEADER.buttons.headerbutton*

**Comment by pkoevesdi** [comment]
This also happens with tags like this:
`<atv:parameter substitute="" behavior="optional" defaultvalue="1" valuetype="number" desc="The bool is considered on, when the value of base is >= onValue" name="onValue"/>`
where the symbol ">" is in a "desc" , not in "id". This tag is produced by atvise builder in the "File" -> "Parameters" section of a display. There, a ">" synbol inside the description ist perfectly ok for atvise builder, but produces this problem with atscm.

**Comment by LukasHechenberger** [comment]
Yes, the parser fails every time an attribute value contains `>`. Thanks for pointing that out!

**Comment by LukasHechenberger** [comment]
Atvise 3.4 is delivered with standard resources that result in this issue (See #355) -> We need a quick fix!

**Comment by LukasHechenberger** [comment]
Fixed as of v1.0.0-beta.48
